target/linux/ath79/dts/qca9557_kt_dw02-412h.dtsi

```diff
@@ -0,0 +1,191 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca955x.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "KT DW02-412H (64M)";
+	compatible = "kt,dw02-412h-64m", "qca,qca9557";
+
+	aliases {
+		led-boot = &led_wan;
+		led-failsafe = &led_wan;
+		led-running = &led_wan;
+		led-upgrade = &led_wan;
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		wps {
+			label = "WPS button";
+			gpios = <&gpio 16 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
+			debounce-interval = <60>;
+		};
+		
+		reset {
+			label = "Reset button";
+			gpios = <&gpio 17 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+			debounce-interval = <60>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_wan: wan {
+			label = "green:wan";
+			gpios = <&gpio 22 GPIO_ACTIVE_LOW>;
+		};
+
+		wifi5g {
+			label = "green:wifi5g";
```

Comment:
On stock FW this LED is used for both 2g and 5g, so strictly "wifi5g" name is wrong IMO.

Reply:
I couldn't find a way to use this one LED for both 2g and 5g. Do you know how to make this work?

Reply:
Well, the name should be what it's used for with the vendor firmware. You should not just pick one, somebody else would probably decide otherwise.
If this is linked to more than one wifi in the OEM firmware, we cannot reproduce that. Best choice is to not set it at all and leave it for the user to decide.

Reply:
That would be the right way, thanks!